### ISSUE TITLE
Only flagging the first character of an erroneous word

### ISSUE THREAD

**Issue by markalfred** [SublimeLinter/SublimeLinter-eslint#231]
Given a file containing simply:

```js
const foo = 'foo';
```

I see the error in the console (no unused vars):
```js
SublimeLinter.plugin.eslint] eslint output:
[{'errorCount': 1,
  'filePath': '/Users/mark/Repos/dtour/web/static/js/lib/financials/foo.js',
  'messages': [{'column': 7,
                'line': 1,
                'message': "'foo' is assigned a value but never used.",
                'nodeType': 'Identifier',
                'ruleId': 'no-unused-vars',
                'severity': 2,
                'source': "const foo = 'foo';"}],
  'source': "const foo = 'foo';\n",
  'warningCount': 0}]
```

And only column 7 is flagged:

![image](https://user-images.githubusercontent.com/3876443/37500365-83b0fca4-289f-11e8-91b0-4a0390a76c95.png)

Expected:

I'd expect the entire word `foo` to be underlined here.

I couldn't find any other issues addressing this — is this working as intended? I find it much more difficult to quickly see where errors exist.

Sorry if: (a) this is some setting that I missed, (b) this is an issue with SublimeLinter itself, or (c) others are *not* having the same issue and it's a result of my config somehow.

**Comment by kaste** [comment]
You must be using a relatively old eslint here. What version are you running?

**Comment by markalfred** [comment]
Good call @kaste, sorry for the wasted time — I'm still on 3.x on this project, upgrading should resolve. Thank you!

**Comment by kaste** [comment]
Nah, v3.x is not old. We should support that.

**Comment by braver** [comment]
Well, a year in JavaScript land is like a century in some other languages ;)

**Comment by kaste** [comment]
Checkout #233

**Comment by kaste** [comment]
🚢
